src/main/java/mat/client/shared/MeasureLibraryResultTable.java

```diff
@@ -4,16 +4,16 @@
 import java.util.LinkedList;
 import java.util.List;
 
+import com.google.gwt.cell.client.*;
+import com.google.gwt.core.client.GWT;
+import com.google.gwt.dom.client.*;
```

Comment:
@adongare  I suggest you should setup your intellij to prefer classes imports vs wild card imports

Reply:
that's good idea. I'll do it. Thanks!

Reply:
as I see from the MAT sources consistently adhere to classes import

Reply:
let me know if you need guidance on that

Reply:
I did not find any way to disable it but I increased **Class count to import with '*'** from 5 to 50. This stops wildcard imports now. remaining settings kept to default(tabs, spaces, indedents, line breaks)